### ISSUE TITLE
Landscape progress bar is involved

### ISSUE THREAD

**Issue by CySxL** [PoomSmart/YTShortsProgress#1]
When YTShortsProgress is installed/enabled, the progress bar in landscape mode of video is always trigger-able/scrollable regardless if it's already hided/invisible or not.

It is annoying and frustrating when I wanna use uYou gesture to adjust volume/brightness, the gesture will scroll on progress bar instead.

Not sure if there is anything can be fixed or it's just how it is, but I'll leave the observation here to help others navigating this issue.

**Comment by PoomSmart** [comment]
@CySxL If you could compile a tweak, try removing this part from the code before building the deb and seeing if that fixes the issue for you

```
%hook YTInlinePlayerBarContainerView

- (void)setUserInteractionEnabled:(BOOL)enabled { %orig(YES); }

%end
```

**Comment by PoomSmart** [comment]
Fixed in 1.0.2

**Comment by CySxL** [comment]
> Fixed in 1.0.2

Confirmed fixed ! You are the legend !
